charts/probate-submit-service/values.aat.template.yaml

```diff
@@ -0,0 +1,6 @@
+java:
+  environment:
+    AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    SERVICES_CORECASEDATA_BASEURL : http://ccd-data-store-api-aat.service.core-compute-aat.internal
+    SPRING_APPLICATION_JSON : {"registries":[{"address":"Digital Application\nOxford District Probate Registry\nCombined Court Building\nSt Aldates\nOxford\nOX1 1LY","email":"test.probate.inbox@gmail.com","id":0,"name":"oxford"},{"address":"Digital Application\nBirmingham Probate Registry\nThe Priory Courts\n33 Bull Street\nBirmingham\nWest Midlands\nB4 6DU\n","email":"test.probate.inbox@gmail.com","id":1,"name":"birmingham"},{"address":"Digital Applications\nManchester District Probate Registry\nCivil Justice Centre\nGround Floor\n1 Bridge Street West\nP.O. BOX 4240\nManchester\nM60 1WJ","email":"test.probate.inbox@gmail.com","id":2,"name":"manchester"}]}
```

Comment:
yeah ... this could be the problem
you can have multi line strings by using yamle  `:|`
```yaml
SPRING_APPLICATION_JSON : |
    "registries":[
        {
            "address":"Digital Application\nOxford District Probate Registry\nCombined Court Building\nSt Aldates\nOxford\nOX1 1LY",
            "email":"test.probate.inbox@gmail.com",
            "id":0,
            "name":"oxford"
        },
        {
            "address":
            "Digital Application\nBirmingham Probate Registry\nThe Priory Courts\n33 Bull Street\nBirmingham\nWest Midlands\nB4 6DU\n",
            "email":"test.probate.inbox@gmail.com",
            "id":1,
             "name":"birmingham"
        },
        {
                "address": "Digital Applications\nManchester District Probate Registry\nCivil Justice Centre\nGround Floor\n1 Bridge Street West\nP.O. BOX 4240\nManchester\nM60 1WJ",
                "email":"test.probate.inbox@gmail.com",
                "id":2,
                "name":"manchester"
         }]
```